x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/EqlSpec.java

```diff
@@ -10,15 +10,17 @@
 import org.elasticsearch.common.Strings;
 
 import java.util.Arrays;
+import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
 public class EqlSpec {
     private String name;
     private String description;
     private String note;
     private String[] tags;
     private String query;
-    private long[] expectedEventIds;
+    private List<long[]> expectedEventIds;
```

Comment:
Please, add a comment to this line as well, mentioning a list of possible valid results to check.